src/Present.hs

```diff
@@ -355,6 +370,20 @@ makeConstructor =
       ((\x y -> [x,y]) <$> makeSlot t1 <*> makeSlot t2)
     (TH.ForallC _ _ con) ->
       makeConstructor con
+#if MIN_VERSION_template_haskell(2,11,0)
+    TH.GadtC names slots _type ->
+      case names of
+        name:_ -> -- FIXME: What about the other names?
```

Comment:
A cleaner solution might be to change the return type of `makeConstructor` to `Either String [Constructor]`. What do you think, @chrisdone?